cdnsystem/daemon/mgr/cdn/storage/storage_mgr.go

```diff
@@ -93,6 +101,8 @@ type PieceMetaRecord struct {
 	PieceStyle  types.PieceFormat `json:"pieceStyle"`  // 1: PlainUnspecified
 }
 
+const fieldSeparator = ":"
```

Comment:
Remove this variable.

Reply:
It is needed to assemble and split strings